sdk/core/azure-core-tracing-opentelemetry/vcpkg/vcpkg.json

```diff
@@ -15,7 +15,7 @@
     {
       "name": "azure-core-cpp",
       "default-features": false,
-      "version>=": "1.7.0-beta.1"
+      "version>=": "1.8.0"
     },
     {
       "name": "opentelemetry-cpp",
```

Comment:
@LarryOsterman, the minimum version of opentelemetry-cpp specified here is "1.3.0". Is that still true?

Reply:
As far as I know, it is still true. I don't believe we've taken a dependency on opentelemetry-cpp code post 1.3.0 (the current version is 1.8.2).